src/webapp/components/current-call/CurrentCallContent.tsx

```diff
@@ -5,7 +5,11 @@ import { useDataSubmissionSteps } from "../../hooks/useDataSubmissionSteps";
 import { DataSubmissionSteps } from "./DataSubmissionSteps";
 import styled from "styled-components";
 
-export const CurrentCallContent: React.FC = () => {
+interface CurrentCallContentProps {
```

Comment:
[minor, best-practices] Props are also typically exported, it may be needed by the parent component to reference a prop type.

Reply:
Best practice update has now been applied to https://github.com/EyeSeeTea/glass-dev/pull/13/commits/683f83b025eae4cb573b2692ef0dfe0bccc59274